docker/ml/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM demisto/python3-deb:3.8.5.11787
+
+COPY requirements.txt .
+
+RUN apt-get update && apt-get install -y    python3-dev g++
+
+RUN pip3 install spacy
```

Comment:
you should not install packages via pip3 in the Dockerfile. This is done via the use of the Pipfile and Pipfile.lock.